src/System.AppContext/ref/System.AppContext.csproj

```diff
@@ -1,6 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
```

Comment:
That shouldn't be a problem. If I'm not mistaken you can override a property at any point so if necessary we could explicitly set CheckForOverflowUnderflow to false for projects that turn out to be problematic. So, should I proceed?